cedar-policy-core/src/extensions/ipaddr.rs

```diff
@@ -100,75 +114,112 @@ impl IPAddr {
         // to true
         // The implementation uses the property that if `ip1/prefix1` is in range
         // `ip2/prefix2`, then `ip1` is in `ip2/prefix2` and `prefix1 >= prefix2`
-        self.addr.is_multicast() && self.subnet >= if self.is_ipv4() { 4 } else { 8 }
+        self.addr.is_multicast() && self.prefix >= if self.is_ipv4() { 4 } else { 8 }
     }
 
     /// Return true if this is contained in the given `IPAddr`
     fn is_in_range(&self, other: &Self) -> bool {
         match (&self.addr, &other.addr) {
             (std::net::IpAddr::V4(self_v4), std::net::IpAddr::V4(other_v4)) => {
-                let netmask = |subnet: u8| u32::MAX.checked_shl(32 - subnet as u32).unwrap_or(0);
-                let hostmask = |subnet: u8| u32::MAX.checked_shr(subnet as u32).unwrap_or(0);
+                let netmask = |prefix: u8| {
+                    u32::MAX
+                        .checked_shl((PREFIX_MAX_LEN_V4 - prefix).into())
+                        .unwrap_or(0)
+                };
+                let hostmask = |prefix: u8| u32::MAX.checked_shr(prefix as u32).unwrap_or(0);
```

Comment:
why do we use `.into()` in one case, but `as u32` in the other case, to convert to `u32`?  I feel like being consistent would make this easier to read